src/test/java/com/stripe/functional/QuoteTest.java

```diff
@@ -0,0 +1,114 @@
+package com.stripe.functional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.exception.StripeException;
+import com.stripe.model.Quote;
+import com.stripe.model.QuoteCollection;
+import com.stripe.net.ApiResource;
+import com.stripe.util.StreamUtils;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class QuoteTest extends BaseStripeTest {
```

Comment:
Do we want to add tests for `Computed`?

Reply:
I've done some testing manually. The [fixture](https://github.com/stripe/openapi/blob/master/openapi/fixtures3.yaml#L1846) that we have from stripe-mock doesn't include line_items, so the regression test wouldn't be a very useful one really.